From ae6e1294935d292452b19e7abf14be0504c12ad5 Mon Sep 17 00:00:00 2001
From: Ta-wei Yen <twyen@google.com>
Date: Mon, 10 Oct 2016 14:43:27 -0700
Subject: [PATCH 2/2] Unexport OmtpMessageReceiver

Introduced in
https://partner-android-review.googlesource.com/#/c/204705/

Receivers do not need to be exported to receive system broadcasts.
OmtpMessageReceiver should also check the intent action, but least
possible scope of change is preferred.

Bug: 31566390
Change-Id: I83f3d42ba185c072357ec6f18b99c3f12f452eb9
(cherry picked from commit a39ff9526aee6f2ea4f6e02412db7b33d486fd7d)
---
 AndroidManifest.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/AndroidManifest.xml b/AndroidManifest.xml
index 0c07e8c..507341c 100644
--- a/AndroidManifest.xml
+++ b/AndroidManifest.xml
@@ -669,7 +669,7 @@
             </intent-filter>
         </provider>
         <receiver android:name="com.android.phone.vvm.omtp.sms.OmtpMessageReceiver"
-            android:exported="true">
+            android:exported="false">
             <intent-filter>
                 <action android:name="android.intent.action.DATA_SMS_RECEIVED" />
                 <data android:scheme="sms" />
-- 
2.9.3

